### ISSUE TITLE
Create a conda recipe

### ISSUE THREAD

**Issue by LuiggiTenorioK** [BSC-ES/autosubmit-api#54]
In GitLab by @kinow on Dec 11, 2023, 14:25

This a sibling issue to the Autosubmit Conda one: https://earth.bsc.es/gitlab/es/autosubmit/-/issues/864

I'm writing it so we don't forget about this, even though it has a low priority, in case it is mentioned in DestinE or EDITO that this would be desired. Or if any progress is made, so that we have somewhere to record information about this issue.

**Comment by LuiggiTenorioK** [comment]
There is an existent conda recipe made by Julian [here](https://earth.bsc.es/gitlab/es/autosubmit_api/-/tree/main/deployment).

I think it still needs to be updated, which I think can be done by just using `grayskull` as it doesn't have complex dependencies as autosubmit :thinking:

**Comment by LuiggiTenorioK** [comment]
In GitLab by @kinow on Dec 11, 2023, 14:57

One of the blockers for Autosubmit is the autosubmit4configparser. I think the same will apply to the autosubmit_api.

**Comment by ntorqulu** [comment]
Doing a quick check, I think AS 4.1.17 (and current 4.2.0) no longer declares nor imports `autosubmit4configparser`.
The `conda-forge` gaps are `autosubmit` itself and `bscearth.utils`, which are not on conda-forge.
If no user asks for it, I would keep the issue open but as low priority.

**Comment by LuiggiTenorioK** [comment]
> If no user asks for it, I would keep the issue open but as low priority.

Agree. Since the creation of this issue, no user has asked for the conda recipe. So, we will keep it as super low priority.

**Comment by kinow** [comment]
Yup. At some point I think we should just get rid of bscearth.utils (there are two projects on pypi, none maintained) by copying the code (<100 lines?) into AS code base, and then unblocking this issue and the AS one.

**Comment by ntorqulu** [comment]
Got it, I'll open an issue in AS to copy the code from bscearth.utils and set it as blocker for this other one [#864](https://github.com/BSC-ES/autosubmit/issues/864). Thanks!
